paymentsheet/src/test/java/com/stripe/android/utils/CompletableSingle.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.utils
+
+import com.stripe.android.common.coroutines.Single
+import kotlinx.coroutines.CompletableDeferred
+
+internal class CompletableSingle<T>(val value: T? = null) : Single<T> {
+    private val deferred: CompletableDeferred<T> = value?.let {
```

Comment:
I'm not entirely sure what the best thing to do here is. And this code isn't super clear to me.